### PR TITLE
Convert Keras Flatten Layer to ONNX Flatten Op

### PR DESCRIPTION
**Pull request by themummy**
The current implementation uses an ONNX Reshape Op, which is causing an issue when converting from ONNX to TF to TFLite.

IMO, using a Flatten Op simplifies the graph's structure, and it's a widely supported Op.

### CODE REVIEW

File: keras2onnx/ke2onnx/main.py [hunk 1]
```diff
@@ -6,32 +6,32 @@
 from collections.abc import Iterable
 
 import numpy as np
-from ..proto import keras, is_tf_keras, is_keras_older_than
```
Comment (themummy) [inline keras2onnx/ke2onnx/main.py]: I didn't change any of the imports, I just changed the order to follow PEP8.

Comment (wenbingl) [reply keras2onnx/ke2onnx/main.py]: We don't enable PEP8 for this project, and the order here is organized by the category.

File: keras2onnx/ke2onnx/main.py [hunk 2]
```diff
@@ -91,23 +91,26 @@ def convert_keras_concat(scope, operator, container):
...
                         operator_name=operator.raw_operator.name + "_transpose", perm=perm)
-        apply_reshape(scope, input_tensor_name, operator.outputs[0].full_name, container,
-                      operator_name=operator.raw_operator.name, desired_shape=target_shape)
+        _apply_flatten(scope, input_tensor_name, operator.outputs[0].full_name, container,
+                       operator_name=operator.raw_operator.name)
+
+
+# TODO: This function should be moved to onnxconverter_common.onnx_ops
+def _apply_flatten(scope, input_name, output_name, container, operator_name=None):
+    from onnxconverter_common.onnx_ops import _create_name_or_use_existing_one
```
Comment (wenbingl) [inline keras2onnx/ke2onnx/main.py]: would you please add apply_flatten in  onnxconverter-common before this PR?

File: keras2onnx/ke2onnx/main.py (hunk 1 above)
Comment (themummy) [reply keras2onnx/ke2onnx/main.py]: OK, I'll revert it back.

File: keras2onnx/ke2onnx/main.py (hunk 2 above)
Comment (themummy) [reply keras2onnx/ke2onnx/main.py]: This PR will take care of it: https://github.com/microsoft/onnxconverter-common/pull/31

File: keras2onnx/ke2onnx/main.py (hunk 1 above)
Comment (themummy) [reply keras2onnx/ke2onnx/main.py]: Done.

File: keras2onnx/ke2onnx/main.py (hunk 2 above)
Comment (jiafatom) [reply keras2onnx/ke2onnx/main.py]: just merged this change, thanks

Comment (themummy) [reply keras2onnx/ke2onnx/main.py]: @jiafatom, thank you!

@wenbingl, do we have to wait until this change makes it into a release?

Comment (wenbingl) [reply keras2onnx/ke2onnx/main.py]: All keras-onnx CI build and tests are depended on the latest master branch of onnxconverter-common, so there is no wait. And we will release onnnxconverter-common before the next release of keras-onnx.

Comment (themummy) [reply keras2onnx/ke2onnx/main.py]: Done.

**Pull request merged**